packages/venia-ui/lib/components/ProductOptions/swatchList.js

```diff
@@ -1,26 +1,38 @@
-import React from 'react';
+import React, { useMemo } from 'react';
 import { arrayOf, func, object, shape, string } from 'prop-types';
-import { List } from '@magento/peregrine';
 import Swatch from './swatch';
 
 import { mergeClasses } from '../../classify';
 import defaultClasses from './swatchList.css';
 
 const SwatchList = props => {
-    const { getItemKey, initialSelection, items, onSelectionChange } = props;
+    const {
+        getItemKey,
+        initialSelection = {},
```

Comment:
I want to change this prop name to `selectedValue`. The time at which the value is selected doesn't matter.